librz/include/rz_type.h

```diff
@@ -197,6 +197,14 @@ typedef enum {
 	RZ_TYPE_COND_LS ///< Unsigned lower or same     Less than or equal
 } RzTypeCond;
 
+/**
+ * \brief Pretty printing options for types
+ */
+typedef enum {
+	RZ_TYPE_PRINT_MULTILINE,
+	RZ_TYPE_PRINT_UNFOLD
```

Comment:
```suggestion
	RZ_TYPE_PRINT_MULTILINE = 1 << 0,
	RZ_TYPE_PRINT_UNFOLD = 1 << 1,
```
Otherwise they can't be easily mixed with `|`

Reply:
I already did this. Just haven't committed yet :)